README.md

```diff
@@ -366,6 +365,41 @@ enable DEBUG logging in the SDK. You can add your own log statements in
 your workflow for further identification of how your workflow script
 is progressing.
 
+## TLS Capability
+
+The websocket server that is built-in to this Python Relay SDK has support
+for providing TLS encryption, given that you have a TLS key and certificate.
+For example, you can get a TLS key and certificate from letsencrypt.org. Or
+you can use a commercial provider. Sometimes this key and certificate for
+HTTP-over-TLS is called an "SSL certificate".
+
+To use the built-in TLS support, the key and certificate both need to
+be in PEM format, and in a file that can be read locally by the workflow
+process.
+
+When not using TLS support, your workflow app should have a line that
```

Comment:
Changes look good!

This comment about "not using TLS support" made me wonder ... have we given guidance anywhere that TLS is a "best practice"?  (Does ibot even require it?)

So "not using TLS support" in the SDK is fine when behind a proxy, etc with TLS support.

Reply:
This is why it's good to have extra eyes, that aren't contaminated with previous education ;-) See if the 3rd commit clarifies this sufficiently.

Reply:
perfect!